### PR TITLE
Implement UserName/Password  to call twin elevated, and set up u/Pw i…

### PR DESCRIPTION
**Pull request by dacolgit**
…n published nodes jobs

Add dataProtection to session stored data

### CODE REVIEW

File: samples/src/Microsoft.Azure.IIoT.App/src/Services/Publisher.cs [hunk 1]
```diff
@@ -68,8 +70,8 @@ public class Publisher {
         /// <param name="samplingInterval"></param>
         /// <param name="publishingInterval"></param>
         /// <returns>ErrorStatus</returns>
-        public async Task<bool> StartPublishingAsync(string endpointId, string nodeId, string displayName,
```
Comment (marcschier) [inline samples/src/Microsoft.Azure.IIoT.App/src/Services/Publisher.cs]: Convention is async methods should end in "Async" for name.   It is ok - but please fix in future again.

File: samples/src/Microsoft.Azure.IIoT.App/src/Models/CredentialModel.cs [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+//  Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
+// ------------------------------------------------------------
+
+using System.ComponentModel.DataAnnotations;
+
+namespace Microsoft.Azure.IIoT.App.Models {
+    public class CredentialModel {
+        [Required]
+        [StringLength(25, MinimumLength = 3, ErrorMessage = "UserName should be between 3 and 25 characters.")]
+        public string Username { get; set; }
```
Comment (cristipogacean) [inline samples/src/Microsoft.Azure.IIoT.App/src/Models/CredentialModel.cs]: I think we should not add any restrictions for username nor for password. you get all kinds of servers  weird in the field ...

File: samples/src/Microsoft.Azure.IIoT.App/src/Pages/Browser.razor [hunk 3]
```diff
@@ -357,7 +361,7 @@
         {
             if (node.Id == samples.NodeId)
             {
-                node.Value = $"{samples.Value?.ToString()} ({samples.Status}/{samples.Timestamp})";
```
Comment (cristipogacean) [inline samples/src/Microsoft.Azure.IIoT.App/src/Pages/Browser.razor]: >/{samples.Timestamp [](start = 76, length = 19)

I think showing the stamp is valuable especially for diagnostics - the user has a feedback when the value been last refreshed.

Comment (marcschier) [reply samples/src/Microsoft.Azure.IIoT.App/src/Pages/Browser.razor]: Oops - overlooked this also.   Please do not undo code during merge, please merge, or ask person making change about context.

File: samples/src/Microsoft.Azure.IIoT.App/src/Services/Publisher.cs (hunk 1 above)
Comment (dacolgit) [reply samples/src/Microsoft.Azure.IIoT.App/src/Services/Publisher.cs]: fixed

File: samples/src/Microsoft.Azure.IIoT.App/src/Models/CredentialModel.cs (hunk 2 above)
Comment (dacolgit) [reply samples/src/Microsoft.Azure.IIoT.App/src/Models/CredentialModel.cs]: ok removed

File: samples/src/Microsoft.Azure.IIoT.App/src/Pages/Browser.razor (hunk 3 above)
Comment (dacolgit) [reply samples/src/Microsoft.Azure.IIoT.App/src/Pages/Browser.razor]: Sorry I didn't realize it was removed during the merge

**Pull request merged**